vnext/Microsoft.ReactNative/Fabric/Composition/Modal/WindowsModalHostViewComponentView.cpp

```diff
@@ -86,6 +86,50 @@ void WindowsModalHostComponentView::EnsureModalCreated() {
     throw std::exception("Failed to create new hwnd for Modal: " + GetLastError());
   }
 
+  // set the top-level windows as the new hwnd
+   winrt::Microsoft::ReactNative::ReactCoreInjection::SetTopLevelWindowId(host.InstanceSettings().Properties(), reinterpret_cast<uint64_t>(m_hwnd));
+
+  // create new compositor - handles the creation/manipulation of visual objects
+  auto compositor = winrt::Windows::UI::Composition::Compositor();
```

Comment:
I'm creating a new compositor here, but is there a recommended way to try get the current compositor used? Do we need if/defs to handle Microsoft vs Windows namespaces or is there a wrapper already made for this?

Following documentation here: https://learn.microsoft.com/en-us/windows/apps/desktop/modernize/using-the-visual-layer-with-win32

Reply:
Something like:

auto compositionContext = winrt::Microsoft::ReactNative::Composition::implementation::CompositionUIService::GetCompositionContext(m_reactContext.Properties());

auto compositor = winrt::Microsoft::ReactNative::Composition::Experimental::MicrosoftCompositionContextHelper::InnerCompositor(
              compositionContext);


I dont think we need to worry about Windows namespace version here anymore, so your code should use the WinUI3 (Microsoft namespace) versions of these things. See CompositionHwndHost::Initialize as an example.